libs/application/templates/parental-leave/src/fields/Review/index.tsx

```diff
@@ -0,0 +1,576 @@
+import React, { FC, useState } from 'react'
+import { useFormContext } from 'react-hook-form'
+import {
+  FieldBaseProps,
+  formatText,
+  getValueViaPath,
+} from '@island.is/application/core'
+import {
+  Accordion,
+  AccordionItem,
+  Box,
+  GridColumn,
+  GridRow,
+  Input,
+  Text,
+} from '@island.is/island-ui/core'
+import {
+  FieldDescription,
```

Comment:
I feel like this file is too big. Maybe we can reuse better some of the components? or split this into more isolated components, one for each accordion item. This is a tad too much

Reply:
Just pushed two refactors:
`PaymentsTable` and `YourRightsBoxChart` which cut out a bunch of repeated code from the Review screen. 
I think we can probably also wrap the periods component...
What do you think?

Reply:
I like this! I think we can do that in a future PR, this is great for now